scripts/ls_public_bucket.py

```diff
@@ -81,7 +81,7 @@ def get_coords(geo_ref_points, spatial_ref):
     t = osr.CoordinateTransformation(spatial_ref, spatial_ref.CloneGeogCS())
 
     def transform(p):
-        lon, lat, z = t.TransformPoint(p['x'], p['y'])
+        lat, lon, z = t.TransformPoint(p['x'], p['y'])
```

Comment:
people might be using this script in GDAL2 env because of "well established" `wget then run pattern` so should probably have a version check here and keep old behaviour for GDAL2